support/build/Makefile.rules

```diff
@@ -269,15 +270,16 @@ define addplatlib_s =
 $(if $(filter y,$(3)),$(call addplatlib,$(1),$(2)),)
 endef
 
+sub_libbuild_dir = $(call sub_build_dir,$(1))/$(2)
```

Comment:
For safety reasons, I'd prefer to keep the `strip` operation on the second parameter

Reply:
You mean `strip` or `notdir`? (latter is in line with `sub_build_dir`)